server/validate/email/validate.go

```diff
@@ -119,6 +121,31 @@ func isTemplateValid(templ *textt.Template) error {
 	}
 	return nil
 }
+type SMPTLogin struct {
+	username, password string
+}
+
+func LoginAuth(username, password string) smtp.Auth {
```

Comment:
I don't think you need to export this method from the module. Consequently, please change the name to have a lowercase first letter. If you believe it needs to be exported, please explain why and add a required godoc comment.

Reply:
Actually, you don't even need this method. You can initialize the struct directly. It's only needed if you are writing a module, but you are not.